src/components/InputBox/index.js

```diff
@@ -1,12 +1,14 @@
+/* eslint-disable react/jsx-no-duplicate-props */
+/* eslint-disable prettier/prettier */
 import React from 'react';
 
 import './Inputbox.scss';
 
-const InputBox = ({ name, placeholder, type, onChangeInput, value }) => {
+const InputBox = ({ name, placeholder, inputtype, onChangeInput, value }) => {
```

Comment:
Input type prop should be in camel casing instead

Reply:
I have fixed this